src/main/resources/application.yaml

```diff
@@ -4,7 +4,7 @@ spring:
   datasource:
     url: jdbc:postgresql://localhost:5432/sintern
     username: postgres
-    password: admin
+    password: P@trici@1
```

Comment:
Please don't expose the your local env. username and password in the commit. This should be left blank.